Apps/JobScheduler/src/Jobs/EmailJob.cs

```diff
@@ -154,6 +132,28 @@ private static MimeMessage PrepareMessage(Email email)
             return msg;
         }
 
+        private void ProcessEmails(List<Email> resendEmails)
```

Comment:
We could name this SendEmails in the 1st place. It is fine to keep this name for now or update it to say what this method exactly does. Btw, does resendEmails param mean that this process only Resend (not Send the 1st time)?

Reply:
This was pre-existing code, I simply fixed the smell with the case error.  If you'd like to have this on the records to change then please go ahead and raise a bug for triage.

Reply:
It is true, I'm good to skip this for now. Whoever gets to this later may have an opportunity to update it then.